core/src/main/java/com/xmartlabs/base/core/module/GsonModule.java

```diff
@@ -22,14 +23,19 @@
 
 import dagger.Module;
 import dagger.Provides;
+import lombok.Setter;
 
 @Module
 public class GsonModule {
   static final String SERVICE_GSON_NAME = "ServiceGson";
 
+  @Setter
+  @Nullable
+  private Function<GsonBuilder, GsonBuilder> modifyGsonBuilder;
```

Comment:
I though that inheritance here would be harder to implement than a simple callback, but i was wrong. Thanks